cluster/gce/util.sh

```diff
@@ -1679,14 +1647,8 @@ function setup-easyrsa {
     cp -r easy-rsa-master/easyrsa3/* easy-rsa-master/kubelet
     mkdir easy-rsa-master/aggregator
     cp -r easy-rsa-master/easyrsa3/* easy-rsa-master/aggregator
-    mkdir easy-rsa-master/konnectivity-server
-    cp -r easy-rsa-master/easyrsa3/* easy-rsa-master/konnectivity-server
-    mkdir easy-rsa-master/konnectivity-agent
-    cp -r easy-rsa-master/easyrsa3/* easy-rsa-master/konnectivity-agent) &>${cert_create_debug_output} || true
```

Comment:
I think you need to keep the `) &>${cert_create_debug_output} || true` since these are part of the `(set -x ...)` function on L1674. There's a bracket mismatch otherwise.

Reply:
Good catch! Updated.